smart_card_connector_app/src/reader-tracker.js

```diff
@@ -20,6 +20,7 @@
  * NaCl (check readerfactory_nacl.cc for more information).
  */
 
+goog.provide('GoogleSmartCard.ReaderInfo')
```

Comment:
The only difference of "public" vs. "local" definitions is that the former ones are assigned to the properties of the global objects like "GoogleSmartCard", while the latter ones are just defined in the file.
Example of a local definition:
`/** @constructor */ function A() {}`